flink-table-store-hive/flink-table-store-hive-catalog/src/main/java/org/apache/flink/table/store/hive/HiveCatalog.java

```diff
@@ -63,6 +64,9 @@
 /** A catalog implementation for Hive. */
 public class HiveCatalog extends AbstractCatalog {
 
+    public static final String INTERNAL_METASTORE_CLIENT_CLASS =
```

Comment:
Maybe we don't need this key.
We can just pass class by the constructor of `HiveCatalog`.